session10/src/Homework/Ex9.java

```diff
@@ -0,0 +1,11 @@
+package Homework;
+
+public class Ex9 {
+
+    public static void main(String[] args) {
+        String binaryNum = Integer.toBinaryString(1011);
```

Comment:
this will return the binary representation of the "normal" (decimal) number 1011 (one thousand and eleven), and not the decimal number of the binary number 1011. So it will return 1111110011 as you already saw. 
What we want instead is to convert this binary 1011 to "normal" number, which is 11. We can do it in this way:
```
String binaryNum = "1011";
int decimalInteger = Integer.valueOf(binaryNum, 2); // 2 means the radix we want to convert from (binary means radix 2)
System.out.println(decimalInteger);
```